src/main/java/org/cloudfoundry/promregator/endpoint/AbstractMetricsEndpoint.java

```diff
@@ -86,7 +86,7 @@
 
 	@Value("${promregator.endpoint.maxProcessingTime:#{null}}")
 	@Deprecated
-	/**
+	/*
```

Comment:
I am a little unsure whether removing the javadoc aspect here is the right thing. In Eclipse, it's no longer shown what shall be used instead...